components/release/release_branch_detail.js

```diff
@@ -122,7 +133,9 @@ function ReleaseBranchDetail(props) {
                 advisoryDetails ?
                     <>
                         <RELEASE_BRANCH_DETAIL_TABLE data={advisoryDetails} currentPage={currentPage} totalPages={totalPages}/>
-                        <Pagination current={currentPage} total={totalPages} onChange={handlePageChange} showSizeChanger={false}/>
+                        <Pagination current={currentPage} pageSize={10} onChange={handlePageChange} showSizeChanger={false}
+                        total={allAdvisories.length * 10} // Multiply by 10 because each "page" is 10 items
```

Comment:
Hmm can I know about these items? Trying to understand the need for a constant here `10`? :thinking:

Reply:
Good catch, I mistakenly mentioned the constant 10 in the context of pagination, this was from first-try implementation and left there since it worked. Upon reviewing the code I realize that it is not necessary. Each page in our pagination represents a single release, and we have one release per page.